documents-d-archeologie-francaise.csl

```diff
@@ -424,214 +406,164 @@
   </citation>
   <bibliography>
     <sort>
-      <key macro="responsability"/>
+      <key macro="author-short"/>
       <key macro="year-date"/>
       <key macro="title"/>
     </sort>
     <layout>
-      <group prefix="" suffix="" delimiter=" ">
+      <group prefix="" suffix="&#9;" delimiter=" ">
```

Comment:
Indeed. It is there to avoid the author and date to be extended all the line long in Word (behavior linked with the block layout?).